Makefile

```diff
@@ -262,10 +262,19 @@ amazon-cluster:
 	aws s3api create-bucket --bucket $(BUCKET_NAME) --region $(AWS_REGION)
 	aws ec2 create-volume --size $(STORAGE_SIZE) --region $(AWS_REGION) --availability-zone $(AWS_AVAILABILITY_ZONE) --volume-type gp2
 
-clean-amazon-cluster:
+amazon-clean-cluster:
```

Comment:
See the new commit. A shell script in the Makefile is used to provide a notice for the user when they delete the AWS pachyderm.